controller/test/test_refugee_controller.py

```diff
@@ -0,0 +1,167 @@
+import unittest
+from datetime import date
+from models.camp import Camp
+from models.refugee import Refugee
+from models.volunteer import Volunteer
+from models.plan import Plan
+from controller.refugee_controller import create_refugee, find_refugee, view_refugee
+
+
+import controller.refugee_controller as rc
```

Comment:
You seem to be importing controller.refugee_controller twice so I think this one can be removed since it's not being used

Reply:
Thanks for the advice. I've deleted the extra ones.